### DISCUSSION TITLE
Flashed unsigned installer into signed firmware via existing openwrt

### DISCUSSION THREAD

**Discussion by RinCat** [dangowrt/owrt-ubi-installer#209] [General]
Try to upgrade to 24.10 which need the pre-release installer. And I flashed unsigned where the device stock firmware was in 1.2. What can I do to fix that? It seems bricked to me.

**Comment by AtssAgency** [comment]
The whole instruction seems to be wrong after 24.10 release. I was only able to install 22.03

**Reply by Jonathon-Doran** [reply]
How do we install 22.03?  I do not see "mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer_signed.itb" anywhere.  Just the unsigned on.
